src/puppet-web/puppet-web.ts

```diff
@@ -353,6 +353,12 @@ export class PuppetWeb extends Puppet {
       }))
     })
 
+    // Sending video files is not allowed to exceed 20MB
+    // https://github.com/Chatie/webwx-app-tracker/blob/master/formatted/webwxApp.js#L1115
+    const videoMaxSize = 20 * 1024 * 1024
+    if (mediatype === 'video' && buffer.length > videoMaxSize)
+      throw new Error('Sending video files is not allowed to exceed 20MB')
```

Comment:
`     throw new Error('Sending video files is not allowed to exceed ' + videoMaxSize/1024/1024 + 'MB')`